src/logging/default-logger.ts

```diff
@@ -55,17 +55,21 @@ export class DefaultLogger implements Logger {
     return '';
   }
 
+  #getPrefix(script: PackageReference) {
+    const label = this.#label(script);
+    return label !== '' ? ` [${label}]` : '';
+  }
+
   log(event: Event) {
     const type = event.type;
-    const label = this.#label(event.script);
-    const prefix = label !== '' ? ` [${label}]` : '';
     switch (type) {
       default: {
         throw new Error(`Unknown event type: ${unreachable(type) as string}`);
       }
 
       case 'success': {
         const reason = event.reason;
+        const prefix = this.#getPrefix(event.script);
```

Comment:
Related to other comment, not sure I understand why we wouldn't always have a `PackageReference` at least.